tests/steps/deploy/k8s/chart/templates/service/deployment.yaml

```diff
@@ -16,7 +16,7 @@ metadata:
     revision: 2ad3293a7675d08bc037ef0846ef55897f38ec8f
   name: dockertest
 spec:
-  replicas: 1
+  replicas: 3
```

Comment:
This is good proof that it works, but perhaps you could also alter one or two of the limit and request fields in the [test_project.yml](https://github.com/Vandebron/mpyl/pull/197/files#diff-cf71122cee41f3d68ad5de7d1cab808c7230c75c448c16bb728c39ccddcc8da0) to see if override and fallback to default work as expected?

Reply:
thanks, indeed there was a mistake in the scaling